CMP1903_A1_2324/Die.cs

```diff
@@ -14,10 +14,16 @@ internal class Die
          */
 
         //Property
-
-
+        public int die_num { get; set; } // establishes get and set for die_num
```

Comment:
Formatting: `die_num` is a public property and should use PascalCase.  I suggest changing to something like `DieNum`.